app/lib/efile/nj/nj_property_tax_eligibility.rb

```diff
@@ -19,26 +19,26 @@ def possibly_eligible_for_deduction_or_credit?(intake)
         end
 
         def determine_eligibility(intake)
-          state_wages = Efile::Nj::NjStateWages.calculate_state_wages(intake)
```

Comment:
[sand] - The reason that `NjStateWages.calculate_state_wages()` is in its own util is because it was called both here and in the calculator, and so was easier to break out for mocking purposes. But if we don't call it here anymore and don't need it for this util's tests, then `NjStateWages.calculate_state_wages()` would only be called in one place in one file... and given that it's only two lines of code anyway, it's now more boilerplate than it's worth.

So I'd say that either in this ticket or a follow-up, we should simplify and get rid of the `NjStateWages` util.